topics/admin/metadata.yaml

```diff
@@ -8,11 +8,10 @@ material:
   -
     title: "Introduction"
     type: "introduction"
+    name: "introduction"
     slides: "yes"
     contributors:
-      -
-        name: "Valentin Marcon"
-        github_username: "ValentinMarcon"
+      - valentinmarcon
```

Comment:
It seems GitHub usernames are not case-sensitive, but maybe we should keep the original spelling here and above.

Reply:
All lower case is better to avoid potential YAML issues (or any future tools that could reuse this content) and for consistency purpose.

Reply:
Fair enough.